### DISCUSSION TITLE
How does prowler maps controls to the tests?

### DISCUSSION THREAD

**Discussion by rezcommltd** [prowler-cloud/prowler#3643] [Q&A]
I was thinking of writing tests and extend prowler for digital ocean because most of our infrastructure is on digital ocean. Can you point me to direction as to where I can find the mappings? for eg: SSH is open and if fixed in digital ocean which controls/control number will be fixed in PCI/HIPAA. How does that map is controlled?

**Comment by toniblyx** [comment]
Hello! First of all, thanks for your interest. We do have a developer guide for you to know more about how to write checks and moreover how to create a new provider (digital ocean) and its services. Actually in Prowler v4 (we will release it tomorrow live [here](https://us06web.zoom.us/webinar/register/WN_EasQzgkCSyCAE6MF2u2BtA#/registration))  adding new providers is going to be pretty straight forward. Join us tomorrow for more information. Also here is our devel guide https://docs.prowler.com/projects/prowler-open-source/en/latest/developer-guide/introduction/

**Comment by gauravpareekk** [comment]
Thank you. However, I'm confused as to how to figure out which control of PCI will be mapped to which tests? Any pointers here?
